src/Compilers/CSharp/Portable/Syntax/LookupPosition.cs

```diff
@@ -32,6 +32,14 @@ internal static bool IsInBlock(int position, BlockSyntax blockOpt)
                 && IsBeforeToken(position, expressionBodyOpt, semicolonToken);
         }
 
+        internal static bool IsInBody(int position, BlockSyntax blockOpt = default(BlockSyntax),
```

Comment:
I think this method should be private and none of the parameters should have default values.